{"uuid": "aecba421-28fa-49ea-b029-2dcf200c777c", "children": ["971fc0ee-fb76-495e-9e0a-d2eb935e2827"], "befores": [{"name": "driver", "status": "passed", "start": 1736559150741, "stop": 1736559151456}], "afters": [{"name": "driver::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=131.0.6778.265)\nStacktrace:\n0   chromedriver                        0x000000010479a138 cxxbridge1$str$ptr + 3653888\n1   chromedriver                        0x0000000104792988 cxxbridge1$str$ptr + 3623248\n2   chromedriver                        0x00000001041f8968 cxxbridge1$string$len + 89228\n3   chromedriver                        0x00000001041d3e44 core::str::slice_error_fail::ha0e52dbcb60e6bae + 3780\n4   chromedriver                        0x0000000104262c84 cxxbridge1$string$len + 524200\n5   chromedriver                        0x0000000104275b60 cxxbridge1$string$len + 601732\n6   chromedriver                        0x0000000104231564 cxxbridge1$string$len + 321672\n7   chromedriver                        0x00000001042321b4 cxxbridge1$string$len + 324824\n8   chromedriver                        0x0000000104764fc0 cxxbridge1$str$ptr + 3436424\n9   chromedriver                        0x00000001047682dc cxxbridge1$str$ptr + 3449508\n10  chromedriver                        0x000000010474be60 cxxbridge1$str$ptr + 3333672\n11  chromedriver                        0x0000000104768b9c cxxbridge1$str$ptr + 3451748\n12  chromedriver                        0x000000010473d678 cxxbridge1$str$ptr + 3274304\n13  chromedriver                        0x00000001047832b4 cxxbridge1$str$ptr + 3560060\n14  chromedriver                        0x0000000104783430 cxxbridge1$str$ptr + 3560440\n15  chromedriver                        0x00000001047925fc cxxbridge1$str$ptr + 3622340\n16  libsystem_pthread.dylib             0x000000019d905f94 _pthread_start + 136\n17  libsystem_pthread.dylib             0x000000019d900d34 thread_start + 8\n\n", "trace": "  File \"/Users/timofeitimko/PycharmProjects/automation_qa_art/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/timofeitimko/PycharmProjects/automation_qa_art/.venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/timofeitimko/PycharmProjects/automation_qa_art/conftest.py\", line 54, in driver\n    attach = driver.get_screenshot_as_png()\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/timofeitimko/PycharmProjects/automation_qa_art/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 839, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/timofeitimko/PycharmProjects/automation_qa_art/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 850, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/timofeitimko/PycharmProjects/automation_qa_art/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/timofeitimko/PycharmProjects/automation_qa_art/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1736559186235, "stop": 1736559186237}], "start": 1736559150741, "stop": 1736559186237}